sentry/templates/hooks/clickhouse-init.job.yaml

```diff
@@ -42,6 +43,17 @@ spec:
           - /bin/bash
           - -ec
           - >-
+            check_readiness() {
+              local host="$1"
+              local port="$2"
+              local count="$3"
+              until [ $count -le 0 ] || clickhouse-client --database=default --host=$host --port=$port --query="{{ $readinessQuery }}" > /dev/null;
+              do
+                echo "Waiting for clickhouse to be ready..."
+                count="$((count-1))"
+                sleep 1
+              done
+            };
             {{- if .Values.clickhouse.enabled }}
             echo "clickhouse-init started"
```

Comment:
Should we not also do a call to `check_readiness` here as well?

Reply:
Yeah, good point. Will do it while rebasing for this https://github.com/sentry-kubernetes/charts/pull/334